src/basics/variable/d.ladziata/homeTask2_Logical_operators/logical_operators

```diff
@@ -0,0 +1,78 @@
+/* 
+Logical OR: Returns expr1 if it can be converted to true; otherwise, returns expr2
+Logical AND: Returns expr1 if it can be converted to false; otherwise, returns expr2
+*/
+
+/*                                                        Task 1                                                 */
+//console.log(NaN || 2 || undefined); //answer : 2
```

Comment:
would be better to add some explanation so why 2 ? because (|| return first true value)